scripts/test/test.sh

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
```

Comment:
So, this guy will replace the run-tests script that we should use when we want to run the tests outside of a full build?